### PR TITLE
feat/1951 Production Login Variables 

### LINKED ISSUE #1951: As tech lead, I need production Login.gov variables scripted in deployments
**Description:**
Per dev sync 7/13, we need to adapt certain Login.gov values from their sandbox environment to their production sites.

**Acceptance Criteria:** 
- [ ] Deployments for production environments are able to use new Login.gov site values
- [ ] Ensure dev/staging deployment behavior are unchanged
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
- [ ] Append Login.gov variables from [here](https://github.com/raft-tech/TANF-app/blob/develop/tdrs-backend/tdpservice/settings/common.py#L342) to our deployment [here](https://github.com/raft-tech/TANF-app/blob/develop/scripts/deploy-backend.sh#L35)
- [ ] Test production deployments are successful using @ADPennington 's fork [here](https://app.circleci.com/pipelines/github/ADPennington/TANF-app)
- [ ] Run Testing Checklist and confirm all tests pass

**Notes:**
- Will need Alex to give assignee permissions in her fork

### PR DESCRIPTION
**Pull request by n0remac**
## Summary of Changes
Pull request closes #1951 

## How to Test
I tested this by adding one of the new variable in CircleCi to purposefully break the app. This shows that the changes I made allow variables in CircleCi to overwrite the default variables.

This shows me setting the variable:

Screen Shot 2022-07-14 at 1 32 33 PM

This confirms the variable was actually set in the sandbox environment:
Screen Shot 2022-07-14 at 1 34 01 PM

This shows the website breaking on login:

https://user-images.githubusercontent.com/7119690/179080392-730fc100-304a-4152-a43d-d177504d0718.mp4

This error message confirms that it was the variable that was added that broke the app:
Screen Shot 2022-07-14 at 1 35 20 PM

### CODE REVIEW

**Comment by n0remac** [comment]
I should revert the test changes we did here.

**Comment by n0remac** [comment]
This is back to normal, but will need a rebase.

File: tdrs-frontend/src/components/SplashPage/SplashPage.jsx [hunk 1]
```diff
@@ -150,27 +150,105 @@ function SplashPage() {
               </p>
             </div>
           </div>
-          <div className="tablet:grid-col-8">
-            <ul className="usa-card-group grid-col-12">
-              <li className="usa-card tablet:grid-col-4 mobile-lg:grid-col-12 resource-card">
+          <div className="usa-accordian__content">
+            <ul className="usa-card-group">
```
Comment (riatzukiza) [inline tdrs-frontend/src/components/SplashPage/SplashPage.jsx:154]: Are these changes meant for this branch?

**Review by ADPennington — APPROVED** [review]
this looks great! thank you @n0remac copying notes from my test below for reference. also, next steps for me is to add these env vars to hhs circleci with production endpoints for login.gov to ensure these overwrite the defaults. 🚀 

**testing notes**:
I manipulated ACR_VALUES as follows: 
1) added an invalid value to CI and upon deployment, trying to login via login.gov redirected me to frontend. never got to a login screen. this value was sent to cloud.gov too. 
2) added the valid value to CI and upon deployment,  it was successful. this value was sent to cloud.gov too
3) removed the env var from CI and cloud.gov and re-deployed. this was also successful, sort of suggesting to me that this value does not need to be set but better to be cautious in production.

**Pull request merged**